test/e2e/provisioning/all_e2e_test.go

```diff
@@ -128,9 +128,11 @@ func TestOpenstackProvisioningE2E(t *testing.T) {
 	osUsername := os.Getenv("OS_USERNAME")
 	osTenant := os.Getenv("OS_TENANT_NAME")
 	osNetwork := os.Getenv("OS_NETWORK_NAME")
+	osInstanceReadyCheckPeriod := os.Getenv("INSTANCE_READY_CHECK_PERIOD")
+	osInstanceReadyCheckTimeout := os.Getenv("INSTANCE_READY_TIMEOUT")
 
-	if osAuthURL == "" || osUsername == "" || osPassword == "" || osDomain == "" || osRegion == "" || osTenant == "" {
-		t.Fatal("unable to run test suite, all of OS_AUTH_URL, OS_USERNAME, OS_PASSOWRD, OS_REGION, OS_TENANT and OS_DOMAIN must be set!")
+	if osAuthURL == "" || osUsername == "" || osPassword == "" || osDomain == "" || osRegion == "" || osTenant == "" || osInstanceReadyCheckPeriod == "" || osInstanceReadyCheckTimeout == "" {
+		t.Fatal("unable to run test suite, all of OS_AUTH_URL, OS_USERNAME, OS_PASSOWRD, OS_REGION, OS_TENANT OS_DOMAIN, INSTANCE_READY_CHECK_PERIOD, and INSTANCE_READY_TIMEOUT must be set!")
```

Comment:
Same here with timeouts defaulting.

Reply:
could you give me a pointer where to add the default values for this variables?